doc/ref_model/chapters/chapter02.md

```diff
@@ -170,8 +170,8 @@ By trying to sort workloads into different categories based on the requirements
 # 2.3 Cloud Infrastructure Profiles
 
 Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
-- **Basic**: for VNFCs that can tolerate resource over-subscription and variable latency.
-- **Network Intensive**: for VNFCs that require predictable computing performance, high network throughput and low network latency.
+- **Basic**: for Workloads [VNFCs/Micro-service] that can tolerate resource over-subscription and variable latency.
```

Comment:
My 2 cents are "Workoads (Pods/VM-s)"

Reply:
@CsatariGergely @pgoyal01 , what about ( Pods/VNFc)

Reply:
My proposal is to omit the brackets completely. We have clear definition of "Workloads" in Glossary, and there is no need to refer to the very specific ETSI NFV term VNFC here.